spring-cloud-dataflow-docs/src/main/asciidoc/task-developer-guide.adoc

```diff
@@ -204,40 +318,30 @@ To do so:
   <artifactId>mariadb-java-client</artifactId>
 </dependency>
 ----
-. From the command line, run `mvn clean package`.
-. From the command line, run `java -jar target/tasklab-0.0.1-SNAPSHOT.jar`.
+. From your command line set up the database connection properties for MySql for example
```

Comment:
We should either say we're using Maria or use the MySql URL in the below config.

Reply:
updated to mysql

Reply:
does it apply to both?

Reply:
While it does, I'd just like us to be consistent.  If we say in the docs we're using MySql...use MySql, etc.